src/main/python/deep_qa/layers/backend/max.py

```diff
@@ -0,0 +1,41 @@
+from keras import backend as K
+from keras.layers import Layer
+from overrides import overrides
+
+from ...tensors.backend import switch, very_negative_like
+
+
+class Max(Layer):
+    '''
+    This Layer performs a max over some dimension.  Keras has a similar layer called
```

Comment:
if "Layer" is capitalized, i feel like you're referring to the object --- in that case, I think this should be in double backticks. If you're just referring to the concept of a layer in general, i think lowercase would be better.